pkg/ddc/jindo/fuse.go

```diff
@@ -0,0 +1,74 @@
+/*
+Copyright 2021 The Fluid Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package jindo
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/fluid-cloudnative/fluid/pkg/common"
+	"github.com/fluid-cloudnative/fluid/pkg/utils"
+	"github.com/pkg/errors"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// cleanupFuse will cleanup node label for Fuse
+func (e *JindoEngine) cleanupFuse() (count int, err error) {
+
+	var (
+		nodeList     = &corev1.NodeList{}
+		fuseLabelKey = common.LabelAnnotationFusePrefix + e.namespace + "-" + e.name
+	)
+
+	labelNames := []string{fuseLabelKey}
+	e.Log.Info("check node labels", "labelNames", labelNames)
+	fuseLabelSelector, err := labels.Parse(fmt.Sprintf("%s=true", fuseLabelKey))
+	if err != nil {
+		return
+	}
+
+	err = e.List(context.TODO(), nodeList, &client.ListOptions{
+		LabelSelector: fuseLabelSelector,
```

Comment:
will delete all nodes with fuse label? how about  other fuse is running on node with lable，will delete lable？thx

Reply:
The selector includes namesapce and name, which must be globally unique for the dataset and bounded jindoruntime. So I think that would not be a problem.